analyses/0012-TUF/README.md

```diff
@@ -0,0 +1,24 @@
+# TUF Documentation Analysis
+
+[Context: "Status: Draft"]
```

Comment:
Actually, this is in quite a good state. I propose that we drop the draft status marker, with the understanding that it's still a live document. @Dindihub 

```suggestion
```